policy_tests/test_groups.py

```diff
@@ -81,7 +81,6 @@ class frtos(AllTests):
                                   "heap-ppac-userType/webapp_patient_info_leak_fails",
                                   "userType/webapp_double_usr_set",
                                   "password/webapp_password_leak",
-                                  "bitcount",
```

Comment:
The frtos group works by taking the tests which are in the allTests group, but which are not in the list here. Bitcount was already in the allTests group, but it was excluded from the frtos group because it was in this list. Since bitcount works on frtos as of this commit, I removed it from the list so that it would be used in the frtos group.